scenarios/academic/KDD2020-tutorial/README.md

```diff
@@ -0,0 +1,51 @@
+# Environment setup
```

Comment:
Do we need to merge this? Or we just leave this in the kdd_tutorial branch?

Reply:
I think we need to keep it, as a quick e2e learning journey.